core-tests/shared/src/test/scala/zio/duration/DurationSpec.scala

```diff
@@ -3,416 +3,320 @@ package zio.duration
+        },
+        test("Make a Duration from positive nanos and check that: min(1 ns, 2 ns) is 1 ns") {
+          assert(Duration.fromNanos(1L).min(Duration.fromNanos(2L)), equalTo(Duration.fromNanos(1L)))
+        },
+        test("Make a Duration from positive nanos and check that: max(2 ns, 1 ns) is 2 ns") {
+          assert(Duration.fromNanos(2L).max(Duration.fromNanos(1L)), equalTo(Duration.fromNanos(2L)))
+        },
+        test("Make a Duration from positive nanos and check that: min(2 ns, 1 ns) is 1 ns") {
+          assert(Duration.fromNanos(2L).min(Duration.fromNanos(1L)), equalTo(Duration.fromNanos(1L)))
+        },
+        test("Make a Duration from positive nanos and check that: 10 ns + 20 ns = 30 ns") {
+          assert(Duration.fromNanos(10L) + Duration.fromNanos(20L), equalTo(Duration.fromNanos(30L)))
+        },
+        test("Make a Duration from positive nanos and check that: 10 ns * NaN = Infinity") {
+          assert(Duration.fromNanos(10L) * Double.NaN, equalTo(Duration.Infinity: Duration))
+        },
+        test("Make a Duration from positive nanos and check that: 10 ns compared to Infinity is -1") {
+          assert(Duration.fromNanos(10L) compare Duration.Infinity, equalTo(-1))
+        },
+        test("Make a Duration from positive nanos and check that: 10 ns compared to 10 ns is 0") {
+          assert(Duration.fromNanos(10L) compare Duration.fromNanos(10L), equalTo(0))
+        },
+        test("Make a Duration from positive nanos and check that: + with positive overflow results in Infinity") {
+          assert(Duration.fromNanos(Long.MaxValue - 1) + Duration.fromNanos(42), equalTo(Duration.Infinity: Duration))
+        },
+        test("Make a Duration from positive nanos and check that: * with negative duration results in zero") {
+          assert(Duration.fromNanos(42) * -7, equalTo(Duration.Zero: Duration))
+        },
+        test("Make a Duration from positive nanos and check that: * with overflow to positive results in Infinity") {
+          assert(Duration.fromNanos(Long.MaxValue) * 3, equalTo(Duration.Infinity: Duration))
+        },
+        test("Make a Duration from positive nanos and check that: * with overflow to negative results in Infinity") {
+          assert(Duration.fromNanos(Long.MaxValue) * 2, equalTo(Duration.Infinity: Duration))
+        },
+        test("Make a Duration from positive nanos and check that: Folding picks up the correct value") {
+          assert(Duration.fromNanos(Long.MaxValue).fold("Infinity", _ => "Finite"), equalTo("Finite"))
+        },
+        test("Make a Duration from negative nanos and check that the Duration is Zero ") {
+          assert(Duration.fromNanos(-1), equalTo(Duration.Zero: Duration))
+        },
+        test("Take Infinity and check that: It returns -1 milliseconds") {
+          assert(Duration.Infinity.toMillis, equalTo(Long.MaxValue / 1000000))
```

Comment:
I realize it was this was in the original but the statement is inconsistent with the code being tested.